tools/yaml-templates/build-test-publish.yml

```diff
@@ -21,8 +21,8 @@ steps:
     displayName: 'Check that changefile was created if needed'
     condition: and(
       eq(variables['Build.Reason'], 'PullRequest'),
-      not(startsWith(variables['System.PullRequest.SourceBranch'], 'release/2.0.0-beta')),
-      not(startsWith(variables['System.PullRequest.TargetBranch'], 'release/2.0.0-beta'))
+      not(startsWith(variables['System.PullRequest.SourceBranch'], 'release/')),
+      not(startsWith(variables['System.PullRequest.TargetBranch'], 'release/'))
       )
     inputs:
       Arguments: 'beachball check'
```

Comment:
My recommendation would be to simply comment out lines 30-37 here to avoid dev releases with inaccurate 2.0.0-beta.# naming.

Reply:
Thanks for pointing this out! I was reasoning over this and I was thinking that until we release v2.0.0, we still want the beta next-dev releases from main. They will continue to query @next-dev and compare with what is in package.json, which will be 2.0.0-beta.7 until we release 2.0. 

Now that I'm typing this out, I'm wondering if that is indeed what we want because we may release @next-dev code changes that aren't present in 2.0.0, but will be present in 2.0.1 (or 2.1, whichever we determine via beachball).

Reply:
Right, I thought that it would be consistent to publish the dev packages as 2.0.1-*.

Reply:
I think it's best to wait until after 2.0.0 to continuing publishing @next-dev releases from `main`. I've disabled the release pipeline and updated this file to skip this step.